src/pages/docs/packaging-applications/create-packages/octopus-cli.md

```diff
@@ -1,79 +1,144 @@
 title: Create packages with the Octopus CLI
-description: Using the Octopus CLI (octo) command line tool to create packages for deployment.
+description: Using the Octopus CLI (octopus) command line tool to create packages for deployment.
 navOrder: 30
 ---
 <!-- spell-checker:ignore Myatt's, PKWARE, Packagingyourapplicationfromafolder -->
-The Octopus CLI (`octo`) is a command line tool that interacts with the [Octopus Deploy REST API](/docs/octopus-rest-api/) and includes a [pack](/docs/octopus-rest-api/octopus-cli/pack) command to create packages either as [Zip](#create-zip-packages) or [NuGet](#create-nuget-packages) packages for deployment with Octopus.
+The Octopus CLI (`octopus`) is a command line tool that interacts with the [Octopus Deploy REST API](/docs/octopus-rest-api/) and includes packaging commands to create packages either as [Zip](#create-zip-packages) or [NuGet](#create-nuget-packages) packages for deployment with Octopus.
 
 ## Installation
 
-The [Octopus CLI downloads page](https://github.com/OctopusDeploy/OctopusCLI/releases) provides installation options for various platforms.
+The [Octopus CLI downloads page](https://github.com/OctopusDeploy/cli/blob/main/README.md#installation) provides installation options for various platforms.
 
 After installation, you can run the following to verify the version of the Octopus CLI that was installed (if you're using Windows, remember to open a new command prompt):
 
 ```
-dotnet octo --version
+octopus --version
 ```
 
-For more installation details, options, and update instructions, see [The Octopus CLI Global Tool](/docs/octopus-rest-api/octopus-cli).
+For more installation details, options, and update instructions, see [The Octopus CLI Global Tool](/docs/octopus-rest-api/cli).
 
-For a full list of the `pack` command options see [Octopus CLI - Pack](/docs/octopus-rest-api/octopus-cli/pack) or run the following command:
+For a full list of the `pack` command options see [Octopus CLI - Package](/docs/octopus-rest-api/cli/octopus-package) or run the following command:
 
 ```powershell
-dotnet octo help pack
+octopus package --help
 ```
 
 ## Usage
 
-At a minimum `octo pack` requires an ID:
+The Octopus CLI supports two package formats: NuGet packages and ZIP packages
+
+### Create NuGet packages {#create-nuget-packages}
+
+At a minimum, `octopus package nuget create` requires an ID:
 
 ```powershell
-dotnet octo pack --id="OctoWeb"
+octopus package nuget create --id="OctoWeb"
 ```
 
 The above command will generate a NuGet package in the current working directory with a time-stamp based version number such as:
 
 > `OctoWeb.2018.6.26.190140.nupkg`.
 
-If you want to provide your own version, you can pass the `--version` parameter in the call to `octo`:
+If you want to provide your own version, you can pass the `--version` parameter:
 
 ```powershell
-dotnet octo pack --id="OctoWeb" --version="1.0.0"
+octopus package nuget create --id="OctoWeb" --version="1.0.0"
 ```
```

Comment:
This section might just be redundant entirely now that the CLI provides so much guidance.